### PR TITLE
V850: Added Extended Floating Point Instructions

### LINKED ISSUE #4453: rh850/e1m-s2 file will not disassemble properly
Hello
I hope you are well.

I am trying to disassemble a Renesas rh850/e1m-s2 processor file  . I am using the v850 language. Most of the file disassembles correctly. There are some instructions that do not disassemble. See the attached code below. Line 8043d66c.

804d334c 40  96  80  00    movhi      0x80 ,r0,r18
        804d3350 f2  99           cmp        r18 ,r19
        804d3352 96  3d           blt        LAB_804d33c4
        804d3354 04  8d           sld.w      0x8 [ep],r17
        804d3356 e0  0f  42  94    cvtf.ws    r1,r18
        804d335a 40  86  80  3f    movhi      0x3f80 ,r0,r16
        804d335e e6  8f  20  74    cmpf.s     le,r17 ,r6,0x0
        804d3362 f0  97  62  84    subf.s     r16 ,r18 ,r16
        804d3366 04  95           sld.w      0x8 [ep],r18
        804d3368 e0  07  00  04    trfsr      0x0
        804d336c f0              ??         F0h
        804d336d 9f              ??         9Fh
        804d336e e0  94           sst.h      r18 ,0xc0 [ep]
        804d3370 b2  05           be         LAB_804d3376
        804d3372 06  55           sld.w      0xc [ep],r10
        804d3374 7f  00           jmp        [lp]

### PR DESCRIPTION
**Pull request by pjsoberoi**
Added fmaf.s, fmsf.s, fnmaf.s fnmfs.s instructions. Closes #4453.

### CODE REVIEW

**Comment by pjsoberoi** [comment]
@GhidorahRex: I added the fmsf.s and fnmaf.s as requested.

**Comment by ryanmkurtz** [comment]
I see you marked that this PR closes #4453, but that ticket is already closed.  Should it be reopened?

**Comment by pjsoberoi** [comment]
@ryanmkurtz, no need to reopen the ticket. User implemented the fix locally and closed the ticket. This is pushing the fix to Ghidra for other users.

File: Ghidra/Processors/V850/data/languages/Instructions/Float.sinc [hunk 1]
```diff
@@ -294,6 +294,31 @@
...
+# FMSF.S reg1, reg2, reg3 - rrrrr111111RRRRR|wwwww10011100010
+:fmsf.s R0004, R1115, R2731 is R0004 & op0510=0x3F & R1115; R2731 & op1626=0b10011100010
+{
+	R2731 = (R1115 f* R0004) f- R2731;
+}
+
+# FNMAF.S reg1, reg2, reg3 - rrrrr111111RRRRR|wwwww10011100100
+:fnmaf.s R0004, R1115, R2731 is R0004 & op0510=0x3F & R1115; R2731 & op1626=0b10011100100
+{
+
```
Comment (incognitojam) [inline Ghidra/Processors/V850/data/languages/Instructions/Float.sinc:312]: There's an extra blank line here

**Review by incognitojam — CHANGES_REQUESTED** [review]

File: Ghidra/Processors/V850/data/languages/Instructions/Float.sinc (hunk 1 above)
Comment (pjsoberoi) [reply Ghidra/Processors/V850/data/languages/Instructions/Float.sinc]: Removed

**Comment by esaulenka** [comment]
Just for your information. Several year ago I did [another sleigh specification](https://github.com/esaulenka/ghidra_v850) for V850.
I am already added these instructions (I think, it is a latest V850 variant, RH850). I am don't have any real binaries for modern Renesas-es, but some users have spoken well about my implementation.

**Comment by pjsoberoi** [comment]
@esaulenka : Any interested in contributing RH850 to Ghidra?

**Pull request closed without merging**